src/jarabe/journal/objectchooser.py

```diff
@@ -160,41 +146,6 @@ def __clear_clicked_cb(self, list_view):
         self._toolbar.clear_query()
 
 
-class TitleBox(VolumesToolbar):
-    __gtype_name__ = 'TitleBox'
-
-    def __init__(self, what_filter='', filter_type=None):
-        VolumesToolbar.__init__(self)
-
-        label = Gtk.Label()
-        title = _('Choose an object')
-        if filter_type == FILTER_TYPE_MIME_BY_ACTIVITY:
-            registry = bundleregistry.get_registry()
-            bundle = registry.get_bundle(what_filter)
-            if bundle is not None:
-                title = _('Choose an object to open with %s activity') % \
-                    bundle.get_name()
```

Comment:
This is a good looking change.  DId you remove the code that sets the title though?